### DISCUSSION TITLE
Creating an 'other taxable income' variable?

### DISCUSSION THREAD

**Discussion by alex-rand** [PolicyEngine/policyengine-canada#84] [General]
This would correspond to the variable `iditoth` in the SPSD/M. From the documentation:

_This variable includes Registered disability savings plan income (Line 125/12500) and other taxable income from Line 130/13000.

The latter includes:
lump-sum payments from pensions and deferred profit-sharing plans, retiring allowances such as an amount paid as severance pay, scholarships, fellowships and bursaries (amounts over $500),
death benefits other than CPP or QPP death benefits._

Not sure if this is worth including? Some of these specific income sources seem to show up in certain benefit calculations. For example, Registered disability savings plan income shows up in the reduction component of the GST credit.

**Comment by MaxGhenis** [comment]
Sounds good, filed https://github.com/PolicyEngine/policyengine-canada/issues/86
